.github/actions/setup-snackpub/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Setup Snackpub
+description: Prepare Snackpub in GitHub Actions
+
+inputs:
+  node-version:
+    description: Version of Node to use
+    default: 16.x
```

Comment:
Is it possible to use Node 18.x since that's the most recent LTS version? (I think Node 16 is fine for this PR, though.)

Reply:
there're still node 16 in other snack services. let's do node upgrade in a separated pr.